### DISCUSSION TITLE
Missing metrics from CrossPlane, but it is mentioned in the documentation

### DISCUSSION THREAD

**Discussion by LHozzan** [crossplane/crossplane#6172] [General]
Hi there.

We using CrossPlane for a while and I looking for some metrics, which can give us some better overview, how the operator working inside. I found another discussions and issues and it seems, that this observability questions are still opened.

Anyway, in [the documentation](https://docs.crossplane.io/v1.17/guides/metrics/) you mentioned metric `crossplane_managed_resource_exists`, which I actually looking for. Unfortunately, when I exposed metrics (enable it in the helm chart values and add PodMonitor object to scrape the metrics endpoint, as is described in the documentation) from our instance, this **metric is missing**.

And here are my confusions:
- Have we something wrong set?
- Is the metric available by default?
- Must be deployed something to have the metric visible?

Thank you!

**Comment by bobh66** [comment]
That metric is generated by the providers, and not all providers may support metrics, so it depends on which provider(s) you are using.  The latest release of provider-upjet-aws has metric support enabled.

**Reply by LHozzan** [reply]
I think, that this is important information.
It will be possible add it to the documentation? I afraid, that I shouldnt be alone, who can be confused.
